src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException  extends Exception {
```

Comment:
Should there be just one space?
```suggestion
public class DukeException extends Exception {
```